### DISCUSSION TITLE
Getting error 500

### DISCUSSION THREAD

**Discussion by Rudiberto** [KatjaGlassConsulting/ApprovalBundle#62] [Q&A] [answered]
Hi, I'm getting this error when starting "Hours Approval"

![image](https://github.com/user-attachments/assets/b58dae29-4945-4e28-a04f-1dba821379b3)

No errors during the installation and creating database.
Here is the corresponding log entry:
`[2024-10-11 14:15:19] [request] CRITICAL: Uncaught PHP Exception TypeError: "DateTime::__construct(): Argument #1 ($datetime) must be of type string, DateTime given" at WeekReportController.php line 133 {"exception":"[object] (TypeError(code: 0): DateTime::__construct(): Argument #1 ($datetime) must be of type string, DateTime given at /kimai/var/plugins/ApprovalBundle/Controller/WeekReportController.php:133)"} []`

**Comment by KatjaGlassConsulting** [comment, marked as answer]
Hi, are you using the latest release: 2.1.1? The 2.1.0 did had this issue and was fixed with 2.1.1.

**Reply by Rudiberto** [reply]
I installed it following your instructions:
`git clone https://github.com/KatjaGlassConsulting/ApprovalBundle ApprovalBundle`
shouldn't that get the actual version?

**Reply by KatjaGlassConsulting** [reply]
You are using Kimai2, right? Could you check in Kimai the plugin-version? It is stated there. Its also available in the downloaded file at composer.json in line 6.

**Reply by Rudiberto** [reply]
So sorry. Just saw that my installation was older than 4 days. I repeated the installation process and everything is working fine now.
Thank you so much.

**Reply by KatjaGlassConsulting** [reply]
Welcome :)
